device/celestica/x86_64-cel_silverstone-r0/Silverstone/minigraph.xml

```diff
@@ -0,0 +1,854 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
no minigraph, use default sku, can you check the latest porting guide.

Reply:
We will remove Minigraph

Reply:
fixed in bf1a418.